core/src/main/java/bio/terra/pearl/core/service/survey/SurveyResponseService.java

```diff
@@ -197,11 +195,12 @@ protected SurveyResponse findOrCreateResponse(ParticipantTask task, Enrollee enr
         } else {
             SurveyResponse newResponse = SurveyResponse.builder()
                     .enrolleeId(enrollee.getId())
-                    .creatingParticipantUserId(participantUserId)
                     .surveyId(survey.getId())
                     .complete(responseDto.isComplete())
                     .resumeData(responseDto.getResumeData())
                     .build();
+            Optional.ofNullable(operator.getParticipantUser()).ifPresent(pUser -> newResponse.setCreatingParticipantUserId(pUser.getId()));
```

Comment:
Hmmm... I like the `setResponsibleUser` on the survey response approach. That mirrors (iirc) the flow on the data audit info. I don't feel super strongly on `if/else`, but I agree it might make the logic clearer.